qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,311 @@
+from .ATS import AlazarTech_ATS
+from .utils import TraceParameter
+from qcodes.utils import validators
+
+
+class AlazarTech_ATS9440(AlazarTech_ATS):
+    """
+    This class is the driver for the ATS9440 board
+    it inherits from the ATS base class
+    """
+    samples_divisor = 256 #32
+
+    def __init__(self, name, **kwargs):
+        dll_path = 'C:\\WINDOWS\\System32\\ATSApi.dll'
+        super().__init__(name, dll_path=dll_path, **kwargs)
+
+        # add parameters
+
+        # ----- Parameters for the configuration of the board -----
+        self.add_parameter(name='clock_source',
+                           parameter_class=TraceParameter,
+                           get_cmd=None,
+                           label='Clock Source',
+                           unit=None,
+                           initial_value='INTERNAL_CLOCK',
+                           val_mapping={'INTERNAL_CLOCK': 1,
+                                               'FAST_EXTERNAL_CLOCK': 2,
+                                               'SLOW_EXTERNAL_CLOCK': 4,
+                                               'EXTERNAL_CLOCK_10MHz_REF': 7})
+        self.add_parameter(name='external_sample_rate',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='External Sample Rate',
+                           unit='S/s',
+                           vals=validators.MultiType(validators.Ints(1000000, 125000000),
+                                                     validators.Enum('UNDEFINED')),
+                           initial_value='UNDEFINED')
+        self.add_parameter(name='sample_rate',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Internal Sample Rate',
+                           unit='S/s',
+                           initial_value=100000000,
+                           val_mapping={1_000: 1,
+                                        2_000: 2,
+                                        5_000: 4,
+                                       10_000: 8,
+                                       20_000: 10,
+                                       50_000: 12,
+                                      100_000: 14,
+                                      200_000: 16,
+                                      500_000: 18,
+                                    1_000_000: 20,
+                                    2_000_000: 24,
+                                    5_000_000: 26,
+                                   10_000_000: 28,
+                                   20_000_000: 30,
+                                   50_000_000: 34,
+                                  100_000_000: 36,
+                                  125_000_000: 38,
+                             'EXTERNAL_CLOCK': 64,
+                                  'UNDEFINED': 'UNDEFINED'})
+        self.add_parameter(name='clock_edge',
```

Comment:
This parameter can be set (it can have set_cmd=None)

Reply:
What do you mean be set? I'm not seeing where the problem is here.

Reply:
Note that for `clock_edge` you do not do `set_cmd=None`, this is done for most other paramters. If you do not do that then `set_cmd=False` which means that you cannot set `clock_edge` only get it.

Reply:
So just add `set_cmd=None` to `clock_edge`?